openupgrade_scripts/scripts/stock_account/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Models in module 'stock_account'---
+---Fields in module 'stock_account'---
+stock_account / account.move.line        / is_anglo_saxon_line (boolean) : DEL
+stock_account / account.move.line        / stock_valuation_layer_ids (one2many): NEW relation: stock.valuation.layer
+stock_account / stock.valuation.layer    / account_move_line_id (many2one): NEW relation: account.move.line
+stock_account / stock.valuation.layer    / price_diff_value (float)      : NEW
+# NOTHING TO DO: new feature
```

Comment:
Shouldn't it be pointed that this would be addresed in `purchase_stock` (I'm not sure if it's necessary/possible/worth it) to rebuild past values